### PR TITLE
AWS- SecurityHub add aws api module 

### PR DESCRIPTION
**Pull request by tallieber**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub.py [hunk 1]
```diff
@@ -3,9 +3,7 @@
 from CommonServerUserPython import *
 
 # flake8: noqa
```
Comment (bakatzir) [inline Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub.py:5]: can rm

File: Packs/AWS-SecurityHub/ReleaseNotes/1_1_0.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+##### AWS - Security Hub
+- Added the Timeout parameter, which enables you to define the amount of time until a timeout exception is reached.
+- Added the Retries parameter, which enables you to define the maximum retry attempts when connection or throttling errors
+    are encountered.
```
Comment (bakatzir) [inline Packs/AWS-SecurityHub/ReleaseNotes/1_1_0.md:6]:
```suggestion
- Added the *Timeout* parameter, which enables you to define the amount of time until a timeout exception is reached.
- Added the *Retries* parameter, which enables you to define the maximum retry attempts when connection or throttling errors are encountered.
```

**Review by bakatzir — COMMENTED** [review]
note that: `Skipping test "AWS-securityhub Test" because it's in skipped test list`
please unskip and make sure that the TPB runs successfully.

**Comment by bakatzir** [comment]
@tallieber FYI for the conflicts you'll have next week due to: https://github.com/demisto/content/pull/12898

**Comment by tallieber** [comment]
> @tallieber FYI for the conflicts you'll have next week due to: #12898

This PR is ready for merge

**Review by bakatzir — APPROVED** [review]

**Pull request merged**